app/helpers/editions_helper.rb

```diff
@@ -0,0 +1,10 @@
+module EditionsHelper
+  def event_action_for_changed_state(state)
+    case state
+    when "ready"
+      "Approved"
```

Comment:
It made more sense to me if it read "Approved by John" rather than "Ready by John". I'm open to suggestions.. "Moved to ready by John"? "Ready for publishing by John"?